### PR TITLE
Added IBM ICD MongoDB

### PR DESCRIPTION
**Pull request by BalramVerma1**
Add IBM ICD MongoDB

### CODE REVIEW

File: filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+data contains details about the query "construct". The construct details the main action (verb) and collections (objects) involved.
+
+The plug-in is free and open-source (Apache 2.0). It can be used as a starting point to develop additional filter plug-ins for the Guardium universal connector.
+
+## 1. Configuring the IBM Cloud MongoDB service
+### Procedure:
+1. Go to https://cloud.ibm.com/login.
+2. On the home page, search and select **Databases for MongoDB**.
+3. Then,  create a database by following the below steps-
+      - In the **Platform** field, select "IBM Cloud".
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md:12]:
```suggestion
      - In the **Platform** field, select **IBM Cloud**.
```

File: filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+         ```sudo yum install -y mongodb-org-shell```
+3. On your deployment's Overview page, there is an Endpoints panel with all the relevant connection information.
+4. Copy the CLI endpoint URL.
+5. Connect to the AWS EC2 instance that you previously created and paste the URL by replacing username (default: admin) and password credentials. The database will be connected.
+
+## 2. Viewing the Audit logs
+### Creating an instance of IBM log analysis on IBM Cloud.
+1. On the home page, search and select **IBM Log Analysis** to create an instance.
+2. Choose your location in the **Location** field.
+3. In the **Pricing Plan** field, select 7 days Log Search.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md:48]:
```suggestion
3. In the **Pricing Plan** field, select **7 days Log Search**.
```

File: filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+* `dropAllRolesFromDatabase`
+* `grantRolesToRole`
+* `revokeRolesFromRole`
+* `grantPrivilegesToRole`
+* `revokePrivilegesFromRole`
+
+## 3. Creating an instance of event streams on IBM Cloud
+### Procedure:
+1. On the home page, search and select **Event Streams** to create the instance.
+2. In the **Platform** field, select "IBM Cloud".
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md:91]:
```suggestion
2. In the **Platform** field, select **IBM Cloud**.
```

File: filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+3. Choose your location in the **Location** field
+4. In the **Pricing plan** field, select Standard.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md:93]:
```suggestion
4. In the **Pricing plan** field, select **Standard**.
```

**Review by AlexaNahum — APPROVED** [review]

File: filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md (hunk 1 above)
Comment (BalramVerma1) [reply filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md]: committed the code as per review comment.

File: filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md (hunk 2 above)
Comment (BalramVerma1) [reply filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md]: committed the code as per review comment.

File: filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md (hunk 3 above)
Comment (BalramVerma1) [reply filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md]: committed the code as per review comment.

File: filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md (hunk 4 above)
Comment (BalramVerma1) [reply filter-plugin/logstash-filter-mongodb-guardium/IBMCloudMongoDB_README.md]: committed the code as per review comment.

**Review by natalygmkibm — APPROVED** [review]

**Review by PratikshaSonawane — APPROVED** [review]

**Pull request merged**